KubeArmor/monitor/logUpdate.go

```diff
@@ -85,6 +85,7 @@ func (mon *SystemMonitor) BuildLogBase(eventID int32, msg ContextCombined) tp.Lo
 	}
 
 	log.Cwd = strings.TrimRight(string(msg.ContextSys.Cwd[:]), "\x00") + "/"
+	log.TTY = strings.TrimRight(string(msg.ContextSys.TTY[:]), "\x00")
```

Comment:
Let's clear the context buffers in ebpf, that might eleminate the need of having this operation

For reference https://github.com/kubearmor/KubeArmor/blob/440bce199dd7e68334d3dddab4c54d579d7a1488/KubeArmor/BPF/shared.h#L292